### PR TITLE
[wip] more perf data collection for jemalloc + rustc

### PR DESCRIPTION
**Pull request by alexcrichton**
Filling in details on https://github.com/rust-lang/rust/issues/36963 ...

### CODE REVIEW

**Comment by alexcrichton** [comment]
@rust-timer build 6073f1b17c83b46986e00adc90c08aed19c6200d

File: src/librustc_driver/lib.rs [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
 #![recursion_limit="256"]
 
+extern crate jemalloc_sys;
```
[the remark this answers is not in the document]
Comment (alexcrichton) [reply src/librustc_driver/lib.rs]: In theory it should be ok here, but let's let the data find out!

**Comment by bjorn3** [comment]
Four benchmarks -10%
One +9%
Most +1-2%

**Comment by alexcrichton** [comment]
@bors: try

Oh I think I measured the wrong thing, that's the exact same as the previous profile

**Comment by alexcrichton** [comment]
@rust-timer build 10c95ccfa7a7adc12f4e608621ca29f9b98eed29

**Comment by alexcrichton** [comment]
Now **that's** what I wanted to see. Gonna close this and will post a more official version soon.

**Pull request closed without merging**